cmd/eval.go

```diff
@@ -252,15 +252,20 @@ func eval(args []string, params evalCommandParams, w io.Writer) (bool, error) {
 		}
 	}
 
-	bs, err := readInputBytes(params)
+	inputBytes, err := readInputBytes(params)
 	if err != nil {
 		return false, err
-	} else if bs != nil {
-		term, err := ast.ParseTerm(string(bs))
+	} else if inputBytes != nil {
+		var input interface{}
+		err := util.Unmarshal(inputBytes, &input)
 		if err != nil {
 			return false, err
```

Comment:
Nit: we might want to wrap the error to indicate it's an error parsing the input as opposed to something else. Would need to try it out manually and see how the error messages look.

Reply:
Good call, I tested it and it does give similar feedback as it was.. but we can do better and say what it is from! Will update.